### PR TITLE
Send a sessionId on graphql requests

### PR DESCRIPTION
**Pull request by tmeasday**
So we can track sessions in the index (PR to follow). The CLI already sends a `sessionId` in a very similar way: https://github.com/chromaui/chromatic-cli/blob/cfb2f280a92b5a01532ca625b0b55b180d8de37b/node-src/index.ts#L48

<details>
  <summary>📦 Published PR as canary version: <code>0.0.24--canary.22.6913426.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromaui/addon-visual-tests@0.0.24--canary.22.6913426.0
  # or 
  yarn add @chromaui/addon-visual-tests@0.0.24--canary.22.6913426.0
  ```
</details>

### CODE REVIEW

File: src/screens/VisualTests/VisualTests.tsx [hunk 1]
```diff
@@ -8,8 +8,8 @@ import { Bar, Col, Row, Section, Sections, Text } from "../../components/layout"
 import { TooltipMenu } from "../../components/TooltipMenu";
 import { getFragment, graphql } from "../../gql";
 import {
-  BuildQuery,
-  BuildQueryVariables,
+  AddonVisualTestsBuildQuery as BuildQuery,
+  AddonVisualTestsBuildQueryVariables as BuildQueryVariables,
```
Comment (weeksling) [inline src/screens/VisualTests/VisualTests.tsx:12]: Is the renaming necessary or should we just be renaming the types below?

**Review by weeksling — APPROVED** [review]

**Comment by weeksling** [comment]
Oh, hey I noticed something odd here... I accepted my own local build on another branch for the no build story, and it seems like it's now being applied as the baseline for merging to main... Is that expected?

File: src/screens/VisualTests/VisualTests.tsx (hunk 1 above)
Comment (tmeasday) [reply src/screens/VisualTests/VisualTests.tsx:12]: The latter I guess, I'll make the change.

**Comment by tmeasday** [comment]
@weeksling it looks like you accepted it on this branch: https://www.chromatic.com/test?appId=6480e1b0042842f149cfd74c&id=64d6994da701c906f8bc611b -- I reached out on slack to try and nail down what happened exactly.

I won't merge this until we figure it out just for record keepin (I think this PR is good to merge though)

**Pull request merged**